File: Selenium/Activities/Activity_12.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("The page title is: ", driver.title)
    print(driver.find_element(By.ID, "input-text").is_enabled())
    driver.find_element(By.ID, "toggleInput").click()
    print(driver.find_element(By.ID, "input-text").is_enabled())
    driver.quit()

File: Selenium/Activities/Activity_12.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_12 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("The pagetitle is: " + driver.getTitle());
        System.out.println(driver.findElement(By.id("input-text")).isEnabled());
        driver.findElement(By.id("toggleInput")).click();
        System.out.println(driver.findElement(By.id("input-text")).isEnabled());
        driver.quit();
    }
}
